content/docs/rfcs/19/README.md

```diff
@@ -0,0 +1,101 @@
+---
+slug: 19
+title: 19/WAKU2-DIRECT-MESSAGE
```

Comment:
Hm, I wonder how we want to distinguish between more application layer and Waku v2 protocols here. For EIPs, there is a different track for ERC, but might be too complex. In any case, I'd put something Ethereum specific here probably? Also feel free to acronym this stuff because it might get too long otherwise (20/WAKU2-APP-ETH-DIRECT-MESSAGE -> WAETHDM? :D probably better ones...)

cc @jm-clius @staheri14 re app/waku v2 level distinction

Reply:
I agree that we need to make explicit the semantic difference between the Waku v2 suite of protocols and any application built on top of it.
To my mind this is ETH-DIRECT-MESSAGE (`EDM` 😎) protocol _over_ Waku v2. Perhaps the spec title can then even omit `WAKU2` and just state its relation to `WAKU2` as part of the spec body? If we want to be more explicit, all such `WAKU2-APP` protocols can be prefixed and referred to as something like `W2A` protocols. Theoretically, `chat2` is already a `W2A` protocol as it operates on a higher layer than `WAKU2`.

Reply:
Oh, I like EDM (well, not the music, too commercial)

Reply:
I went for "ETH-DM" for now. Reasoning:
- Removed `WAKU2` as suggested, keeping this for waku protocol level specs only
- "DM" is a common acronym for "Direct Message",
- "ETH" is a common acronym for "Ethereum".

Reply:
ETH is typically used to refer to the actual coin, Ether. Eth to refer to Ethereum (see for example usage of Eth1 and Eth2). So in that sence, perhaps EDM was a better choice :)